samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -834,7 +834,7 @@ mfxStatus CEncodingPipeline::CreateHWDevice()
     MSDK_CHECK_STATUS(sts, "m_hwdev->Init failed");
 
 #elif LIBVA_SUPPORT
-    m_hwdev = CreateVAAPIDevice();
+    m_hwdev = CreateVAAPIDevice(MFX_LIBVA_DRM, m_strDevicePath.c_str());
```

Comment:
MFX_LIBVA_DRM hardcode **here** seems incorrect place.

Reply:
I just get default value, which found in c`tor